ads/google/a4a/utils.js

```diff
@@ -199,6 +212,8 @@ export function googleBlockParameters(a4a, opt_experimentIds) {
     'ady': Math.round(slotRect.top),
     'oid': '2',
     'act': enclosingContainers.length ? enclosingContainers.join() : null,
+    // aexp URL param is separated by `!`, not `,`.
+    'aexp': aexps ? aexps.replace(/,/g, '!') : null,
```

Comment:
is it a typo? aexp vs aexps?

Reply:
not a typo, the param is called "aexp" internally.

Reply:
ok, i suggest we make variable name the same to avoid mistakes.

Reply:
Done.